src/module_EARTH_GRID_COMP.F90

```diff
@@ -3596,8 +3599,9 @@ subroutine SetModelServices(driver, rc)
               file=__FILE__, rcToReturn=rc)
             return  ! bail out
 #endif
-          elseif (trim(model) == "datm") then
-#ifdef FRONT_DATM
+          elseif (trim(model) == "datm" .or. trim(model) == "datm_cfsr" .or. &
```

Comment:
Are these CDEPS data atmosphere? I think this implementation is not flexible enough because you need to add a line for each specific datm stream. At this level model only need to know the datm not the stream used in CDEPS. So, only datm is enough for driver at this point. Please refer to following version of the file found in https://github.com/hafs-community/NEMS/pull/3

Reply:
Yes, we need to remove `datm_cfsr` and `datm_gefs` because the way to configure the DATM input data is via its namelist.  Moreover, the driver does not need to know what the source of the data is.  Also, this approach means we'd need to change the driver to add a new data source to CDEPS.  All good reasons why we only need `DATM` at the driver level.  This is important to get right.

Reply:
@rsdunlapiv So you're saying that nems.configure will have the atm model either datm (=nemsdatm), datm_cfsr (=cdeps,cfsr source) or datm_gefs (cdeps,gefs source). There is nothing in the driver it self that is needed to distinguish those options.